conf/ocsci/vsphere_upi.yaml

```diff
@@ -2,6 +2,6 @@
 ENV_DATA:
   platform: 'vsphere'
   deployment_type: 'upi'
-  worker_replicas: 3
+  worker_replicas: 6
```

Comment:
Added WIP to really prevent this to be merged. I guess it's just for testing purpose, right? If not you will need to create another file with this change.

Reply:
its just for testing, yes